test-suite/metaschema-xspec/metaschema-test-helper.xsl

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0"
+    xmlns:m="http://csrc.nist.gov/ns/oscal/metaschema/1.0"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    exclude-result-prefixes="#all"
+    xpath-default-namespace="http://csrc.nist.gov/ns/oscal/metaschema/1.0"
+    default-mode="scrubbing">
+
+    <!--
+
+    
+    XSLT provides utility functionality for testing OSCAL profile resolution.
+    
+    Function opr:scrub-document returns an OSCAL catalog or profile with 'cosmetic' whitespace removed,
+    for clean comparison.
+    
+    -->
+
+    <xsl:output method="xml"/>
+    
+    <!--<xsl:strip-space
+        elements="catalog metadata param control group back-matter prop link part usage constraint guideline select remarks description test revisions revision role location party responsible-party address external-id resource citation rlink base64"/>-->
+
+
+    <!--All assemblies defined in the catalog model:
+        
+        https://raw.githubusercontent.com/wendellpiez/OSCAL/profile-resolution-update-v1a/xml/schema/oscal_catalog_schema.xsd
+        
+        //xs:element[exists(xs:complexType[not(@mixed='true')])]/@name => distinct-values() => string-join(' | ')
+        
+        
+        catalog | metadata | param | control | group | back-matter | prop | link | part | usage | constraint | guideline | select | remarks | description | test | revisions | revision | role | location | party | responsible-party | address | external-id | resource | citation | rlink | base64
+        
+        
+    -->
+    <!--<xsl:template match="/">
+        <xsl:param name="source" select="." as="document-node()"/>
+        <xsl:document>
+          <xsl:apply-templates select="$source" mode="scrubbing"/>
+        </xsl:document>
+    </xsl:template>-->
```

Comment:
Minor: should we not just delete this because it is dead code?

```suggestion
```

Reply:
Weird I am not sure why it gets highlighted to be like it is not commented code in the suggestion diff, but it is! haha